client/src/components/ResultsLayout/ResultsLayout.jsx

```diff
@@ -10,13 +9,13 @@ const ResultsLayout = ({ recipes, title, page }) => {
   return (
     resultsLoaded && (
       <div className={clsx("wrapper")}>
-        <h2 className={styles.title}>{title}</h2>
+        <h1 className={styles.title}>{title}</h1>
         {!recipes || recipes.length === 0 ? (
           <EmptyPageContent page={page} />
         ) : (
           <div className={styles.recipesContainer}>
-            {recipes.map((recipe) => (
-              <RecipeCard key={recipe.id} recipe={recipe} page={page} />
+            {recipes.map((recipe, index) => (
+              <RecipeCard key={index} recipe={recipe} page={page} />
```

Comment:
Why did you move away from the recipe ID as key?

Reply:
Oh good catch! I had intended to do this for ingredients when I was still using the list, but think I accidentally updated it in recipes too. I will update this to keep it by id.